src/components/footer/full.njk

```diff
@@ -0,0 +1,137 @@
+---
+layout: layout-example.njk
+---
+
+{% from 'footer/macro.njk' import govukFooter %}
+
+{{ govukFooter({
+  "navigation": [
+    {
+      "title": "Services and information",
+      "columns": 2,
+      "items": [
+        {
+          "href": "/browse/benefits",
```

Comment:
Suggest changing all these links to `'#'` to avoid users getting 404s if they click them.